### ISSUE TITLE
Floating windows with background blur enabled breaks window borders.

### ISSUE THREAD

**Issue by aspauldingcode** [WillPower3309/swayfx#267]
### Please read the following before submitting:
- Proprietary graphics drivers, including nvidia, are not supported. Please use the open source equivalents, such as nouveau, if you would like to use Sway.
- Please do NOT submit issues for information from the github wiki. The github wiki is community maintained and therefore may contain outdated information, scripts that don't work or obsolete workarounds.
  If you fix a script or find outdated information, don't hesitate to adjust the wiki page.

### Please fill out the following:
- **Swayfx Version:**
  - `sway -v` I am not at my desktop ATM. I use the latest unstable NixOS channel, using whichever swayfx version is packaged in the nixpkgs repository as of 01/13/24. 

- **Debug Log:**
  - Run `sway -d 2> ~/sway.log` from a TTY and upload it to a pastebin, such as gist.github.com.
  - This will record information about sway's activity. Please try to keep the reproduction as brief as possible and exit sway.
  - Attach the **full** file, do not truncate it.

- **Configuration File:**
  - Please try to produce with the default configuration.
  - If you cannot reproduce with the default configuration, please try to find the minimal configuration to reproduce.
  - Upload the config to a pastebin such as gist.github.com.

- **Stack Trace:**
  - This is only needed if sway crashes.
  - If you use systemd, you should be able to open the coredump of the most recent crash with gdb with
    `coredumpctl gdb sway` and then `bt full` to obtain the stack trace.
  - If the lines mentioning sway or wlroots have `??` for the location, your binaries were built without debug symbols. Please compile both sway and wlroots from source and try to reproduce.

- **Description:**
  - The steps you took in plain English to reproduce the problem.
Disable background blur.
Toggle Float on Alacritty Terminal (v13)
Notice the window border. it's still there. Good!

Enable swayfx background blur.
Toggle float on Alacritty Terminal (v13)
Notice the window border. It's gone on floating window, but when it is toggled as tiling layout, border is back. Bad news!

**Comment by Minda1975** [comment]
I can't reproduce this in Nixos 23.11 with foot and terminator terminals.

**Comment by aspauldingcode** [comment]
It's definitely still happening to me

**Comment by Minda1975** [comment]
> It's definitely still happening to me

What distro?

**Comment by aspauldingcode** [comment]
NixOS 24.05 (Uakari)
Linux 6.7.3

Here's uname:

```
uname -a
Linux NIXSTATION64 6.7.3 #1-NixOS SMP PREEMPT_DYNAMIC Thu Feb  1 00:21:21 UTC 2024 x86_64 GNU/Linux
```

**Comment by aspauldingcode** [comment]
Here is a gif of my problem:
![2024-02-08 10-03-54](https://github.com/WillPower3309/swayfx/assets/10196826/f5fe4ce7-312c-40ed-8a1e-4d2493cafdd5)

**Comment by Minda1975** [comment]
> NixOS 24.05 (Uakari) Linux 6.7.3

> I thought I was on latest NixOS. my bad.

The latest stable version od NixOS is 23.11. You running unstable version. It is possible that an unstable version of the distribution is the problem of this bug. Hard to say...

**Comment by aspauldingcode** [comment]
I technically fixed it with a brute-force config approach:

```
      #for all windows, brute-force use of "pixel"
      for_window [shell="xdg_shell"] border pixel 2
      for_window [shell="xwayland"] border pixel 2
```
